pkg/volume/csi/csi_mounter_test.go

```diff
@@ -725,6 +809,9 @@ func TestMounterSetUpWithFSGroup(t *testing.T) {
 		}
 
 		csiMounter := mounter.(*csiMountMgr)
+		if tc.driverFSGroupPolicy {
+			csiMounter.fsGroupPolicy = tc.supportMode
+		}
```

Comment:
A note for a follow up cleanup - these tests are not very reliable .. they merely run `Setup` but exact behaviour is not verified..